src/main/java/net/sf/mzmine/util/components/CombinedXICComponent.java

```diff
@@ -76,9 +76,11 @@ public CombinedXICComponent(Feature[] peaks, int id) {
             maxIntensity = Math.max(maxIntensity,
                     peak.getRawDataPointsIntensityRange().upperEndpoint());
             if (rtRange == null)
-                rtRange = peak.getDataFile().getDataRTRange();
+//                rtRange = peak.getDataFile().getDataRTRange();
+                rtRange = peak.getRawDataPointsRTRange();
```

Comment:
Please be careful about making changes to existing MZmine 2 modules as part of your pull requests. Such changes need to be carefully considered and discussed case-by-case. This change, for example, is unacceptable for me - it changes the existing plots into something that is much less useful (in my view). We can discuss what you are trying to achieve somewhere else, but in any case it does not belong into this PR.

Reply:
I'm sorry, we used the latter option in our lab, but you're right I should have discussed it with you before submitting these changes. Please, let me know where we can discuss it and I'll be glad to explain my reasoning.

Reply:
We can discuss by email or you can file an issue here or make a separate PR just for this change, whichever works best for you.